### DISCUSSION TITLE
Request: consider adding Duration scalar.

### DISCUSSION THREAD

**Discussion by bovinemagnet** [graphql-java/graphql-java#3524] [Ideas]
Hi, I was going to add this to the issues, but I saw that it was best to discuss it here first.

Please consider adding a `DurationScalar`  for `java.time.Duration`  as it would definitely be a useful input scalar. 

 And probably a useful output scalar too.
 
Secondly,  If I end up doing it myself, would it be easy to have it added back here ?

thanks

**Comment by bbakerman** [comment]
We would be all for a contribution - it would be to https://github.com/graphql-java/graphql-java-extended-scalars

We would love to see a design of the inputs and outputs of such a scalar.

eg what standard would be followed?  could  sone one in another language implement such input ?
